lib/bundler/cli/plugin.rb

```diff
@@ -5,14 +5,16 @@ module Bundler
   class CLI::Plugin < Thor
     desc "install PLUGINS", "Install the plugin from the source"
     long_desc <<-D
-      Install plugins either from the rubygems source provided (with --source option) or from a git source provided with (--git option). If no sources are provided, it uses Gem.sources
+      Install plugins either from the rubygems source provided (with --source option) or from a remote/git source provided (with respectively --git and --local-git options). If no sources are provided, it uses Gem.sources
    D
     method_option "source", :type => :string, :default => nil, :banner =>
       "URL of the RubyGems source to fetch the plugin from"
     method_option "version", :type => :string, :default => nil, :banner =>
       "The version of the plugin to fetch"
     method_option "git", :type => :string, :default => nil, :banner =>
       "URL of the git repo to fetch from"
+    method_option "local-git", :type => :string, :default => nil, :banner =>
```

Comment:
One thought: if we made this instead `local_git` we can avoid the quotes in `options[:'local-hash']`

Reply:
The rationale for using `local-git` rather than `local_git` was that, in absence of Thor cosmetic automations (I'm not aware of any way to automatically convert hyphen to underscore, aside aliasing), between presenting a non-standard user-facing style (Unix parameters typically use hyphen) and a non-standard programmtic style, the priority was for the user.

Note that at a brief search (`method_option.*_`), I can't find any other method using an underscore style (eg. see `lib/bundler/cli.rb`).

I've pushed the change, anyway :smile:.